recipe/meta.yaml

```diff
@@ -30,21 +32,37 @@ outputs:
         - python
         - pip
         - setuptools
+        - wheel
       run:
         - python
+
+    {% set ignore_tests_files = "" %}
+    # ERROR tests/prof/perf - ModuleNotFoundError: No module named 'benchmarks'
+    {% set ignore_tests_files = ignore_tests_files + " --ignore=tests/prof" %}
+    {% set ignore_tests_files = ignore_tests_files + " --ignore=tests/typing/test_pyright.py" %}
+
+    # Skiping tests [not (win and py==310) and not (win and py==311)]
+    # Windows fatal exception: stack overflow. To many tests to skip one by one
+
     test:
       imports:
         - msgspec
       requires:
         - pip
         - pytest
+        - mypy
+        - attrs
+        - pyyaml
+        - tomli    # [py<311]
+        - tomli-w
+        - msgpack-python
       source_files:
         - tests
```

Comment:
```suggestion
        - tests 
        # benchmarks are here for running the tests suite.
        - benchmarks
```
It will solve `ModuleNotFoundError: No module named 'benchmarks'`

Reply:
It's a folder of the package itself https://github.com/jcrist/msgspec/tree/0.20.0/benchmarks